packages/wdio-logger/src/node.ts

```diff
@@ -111,6 +112,14 @@ const wdioLoggerMethodFactory = function (this: log.Logger, methodName: log.LogL
     }
 }
 
+const progress = function (this: any, data: string) {
+    const level = 'progress'
+    const timestampFormatter = chalk.gray(new Date().toISOString())
+    const levelFormatter = chalk[COLORS[level]](level.toUpperCase())
+    const nameFormatter = chalk.whiteBright(this.name)
+    process.stdout.write(`${timestampFormatter} ${levelFormatter} ${nameFormatter}: ${data}\r`)
```

Comment:
What happens if WebdriverIO writes to a log file, e.g. if `outputDir` is set?

Reply:
When the `outputDir` is configured, the progress will continue to be displayed in the terminal but won't be recorded in the log files.

Reply:
We should treat this custom log level similar to `INFO` which is why I would suggest to check the loglevel in here and don't print anything if loglevel is e.g. `ERROR`